src/app/shared/components/plh-main-header.ts

```diff
@@ -59,15 +61,24 @@ export class PLHMainHeaderComponent implements OnInit, OnDestroy {
    */
   handleRouteChange() {
     // As component sits outside main ion-router-outlet need to access via firstChild method
-    // if wanting to access route params directly (not currently required)
-    const HOME_ROUTE = "/home";
-    // track if home page, allowing case where hosted from subdirectory (e.g. our pr preview system)
-    this.isHomePage = location.pathname.endsWith(HOME_ROUTE);
-    this.isSettingsPage = location.pathname.endsWith("/settings");
+    // if wanting to access route params directly (not currently required)4
+    const HOME_ROUTE = "/home_screen";
+
+    if (this.backHistory.length > 0) {
+      this.isHomePage = false;
```

Comment:
I'm not sure if this will work as intended. If a user goes `Page1 -> Page2 -> Page3 -> Page1` we still have a backHistory but are on the home page, so the check below will say homePage `false`. As well as clicking back buttons we have times when the user might be redirected back to the home page automatically, e.g. after they finish a workshop.

I think we should avoid trying to handle the full backHistory as it becomes quite complicated, and just keep a simple check for whether we are on the home page to show the hamburger menu and keep back button everywhere else